charts/allure/templates/deployment-api.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "allure.fullname" . }}-api
+  labels:
+    {{- include "allure-api.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
Could we split management of replicas of API and UI? so we can increase one or the other individually (`.Values.api.replicaCount` and `.Values.ui.replicaCount`)